test_scripts/Defects/5_0/2482_UNSUPPORTED_RESOURCE_response_is_received_for_SetGlobalProperties_RPC_when_KEYPRESS_MODE_is_set_to_SINGLE_KEYPRESS_or_QUEUE_KEYPRESSES_or_RESEND_CURRENT_ENTRY.lua

```diff
@@ -0,0 +1,165 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/2482
+--
+-- Description:
+-- UNSUPPORTED_RESOURCE response is received for SetGlobalProperties RPC when KEYPRESS MODE is set to SINGLE_KEYPRESS,
+-- QUEUE_KEYPRESSES or RESEND_CURRENT_ENTRY
+-- Precondition:
+-- Vehicle ignition is ON
+-- SYNC is ON
+-- Device is connected to the SYNC via BT
+-- Autotester is installed and registered to the system
+-- In case:
+-- 1) Launch Emergency on the phone
+-- 2) Select Apps from the status interaction menu
+-- 3) Select Emergency app
+-- 4) Select "send message" and "SetGlobalproperties" ,
+-- 	  edit Keyboard properties "KeypressMode as Single_keyPress" and press ok
+-- 5) Select 'Send Message' in the test app on the phone and
+--    Select CreateInteractionChoiceSet then enter all the details and press OK.
+-- 6) Select 'Send Message' in the test app on the phone and
+--    Select "PerformInteraction", Enter data for Initial TextInitial Prompt:
+--    Interaction Mode: MANUAL_ONLY, Layout: KEYBOARD(as in step 4 KeyPress mode: SINGLE_KEYPRESS )and press OK
+-- 7) Verify the Screen displayed on SYNC
+-- 8) Press on the text box and enter the Single key in the keyboard displayed on the sync
+-- Expected result:
+-- 1) SUCCESS response should be recieved for both SetGlobalProperties and PerformInteraction RPC.
+-- Actual result:
+-- UNSUPPORTED_RESOURCE response is recieved fot SetGlobalProperties.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+local keyPressMode = {
+	"SINGLE_KEYPRESS",
+	"QUEUE_KEYPRESSES",
+	"RESEND_CURRENT_ENTRY"
+}
+
+local paramsSetGlobalProp = {
+	helpPrompt = {
+		{
+			text = "Help prompt",
+			type = "TEXT"
+		}
+	},
+	keyboardProperties = {
+		keypressMode = pKeypressMode,
+		limitedCharacterList = {"a"},
+	}
+}
+
+local responseUiParams = {
+	keyboardProperties = paramsSetGlobalProp.keyboardProperties
+}
+
+local responseTtsParams = {
+	helpPrompt = paramsSetGlobalProp.helpPrompt
+}
+
+local performParams = {
+  initialText = "TextInitial",
+  interactionMode = "MANUAL_ONLY",
+  interactionChoiceSetIDList = { 100 },
+  initialPrompt = {
+    { type = "TEXT", text = "pathToFile1" }
+  },
+  helpPrompt = {
+    { type = "TEXT", text = "pathToFile2" }
+  },
+  timeoutPrompt = {
+    { type = "TEXT", text = "pathToFile3" }
+  },
+  interactionLayout = "KEYBOARD"
+}
+
+local performHmiParams = {
```

Comment:
There is not the need to define performHmiParams, values for check in sendPerformInteraction_SUCCESS  can be used from performParams.